lib/kuzzle/kuzzle.js

```diff
@@ -173,8 +173,12 @@ class Kuzzle extends KuzzleEventEmitter {
 
       await this.validation.curateSpecification();
 
+      await this.internalIndex.updateMapping('users', options.import.userMappings);
```

Comment:
What if the mappings are null?

Reply:
They can't, there is an [object type check](https://github.com/kuzzleio/kuzzle/pull/2106/files#diff-aa04b68e2a0c83aa37ee408b049c460ca78e2bda7e4da0b439c13cfaf0500311R52) when the user imports mappings in first place

Reply:
Yes but if none mappings are set by the developer ?